internal/host/host.go

```diff
@@ -728,6 +736,31 @@ func (m *Manager) ReportValidationFailedMetrics(ctx context.Context, h *models.H
 	return nil
 }
 
+func (m *Manager) reportValidationStopSucceedingMetrics(ctx context.Context, vc *validationContext, h *models.Host, newValidationRes map[string][]validationResult) error {
```

Comment:
If it handle both cases please rename. e.g.
`reportValidationStatusChanged`

Reply:
done.